src/main/resources/main.css

```diff
@@ -822,3 +822,11 @@ VBox.spaced, HBox.spaced {
 .error-textfield {
     -fx-effect: dropshadow(three-pass-box, red, 8, 0.0, 0, 1);
 }
+
+.selectable-text * {
```

Comment:
Works but in html this would be very expensive because it selects from right to left and would select every element and then check its perent recursively. Not sure how javafx does this but I would prefer to replace `*` by the exact element if you can figure that out. [Scenic View](http://fxexperience.com/scenic-view/help/) might be of help...